src/Common/src/System/Text/OSEncoding.Windows.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+#nullable enable
```

Comment:
We had convention in corelib to have empty row after license note and no row after `#nullable enable` like:

```
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license
// See the LICENSE file in the project root for more information.

#nullable enable
using System;
...
```